instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/__init__.py

```diff
@@ -300,9 +411,45 @@ def _instrument(self, **kwargs):
             )
 
     def _uninstrument(self, **kwargs):
+        grpc._client = self._original_client
+        _InstrumentedClient._request_hook = None
+        _InstrumentedClient._response_hook = None
+        _InstrumentedClient._tracer_provider = None
         for ctype in self._which_channel(kwargs):
             unwrap(grpc, ctype)
-
+        
+    @staticmethod
+    def instrument_client(
```

Comment:
The scope of the issue is to add request/response hooks only. I don't know what you had mind but these changes don't seem to be relevant to me.